jest.config.js

```diff
@@ -0,0 +1,32 @@
+module.exports = {
+  verbose: true,
+  collectCoverage: true,
+  setupTestFrameworkScriptFile: '<rootDir>/src/tests/setupTests.js',
+  coverageReporters: ['text', 'lcov'],
+  collectCoverageFrom: [
+    '**/*.{js,jsx}',
+    '!**/*.{config}.{js}',
+    '!**/node_modules/**',
+    '!**/dist/**',
+    '!**/vendor/**',
+    '!**/coverage/**',
+    '!**/src/assets/**',
+    '!**/semantic/**',
+    '!**/src/index.js',
+    '!**/cypress/**',
+    '!**/src/components/App.js',
+    '!**/src/config.js'
+  ],
+  testPathIgnorePatterns: [
+    '<rootDir>/cypress/',
```

Comment:
See above.

Do you see a reason for having cypress both in the 'don't run tests' and 'don't generate coverage' sections?

Reply:
The former tells jest where to collect coverage from(we do not want it collecting coverate for cypress tests) while the latter tells jest to ignore running tests in files within the cypress directory as they should be run using cypress command.

Reply:
That's clear from the code. What I mean to ask is, is there a good reason to explicitly say "don't generate coverage" for tests that are never being run through jest?